File: bidly-backend/src/main/java/com/bidly/bidly_backend/model/Duenio.java
package com.bidly.bidly_backend.model;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "duenios")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Duenio {

    @Id
    private Long identificador;

    @Column(name = "numeropais")
    private Integer numeroPais;

    @Column(name = "verificacionfinanciera")
    private String verificacionFinanciera;

    @Column(name = "verificacionjudicial")
    private String verificacionJudicial;

    @Column(name = "calificacionriesgo")
    private Integer calificacionRiesgo;

    @Column(name = "verificador", nullable = false)
    private Long verificador;
}

File: bidly-fastapi/app/models/duenio.py
from sqlalchemy import Column, Integer, String, ForeignKey
from app.database import Base


class Duenio(Base):
    __tablename__ = "duenios"

    identificador          = Column(Integer, ForeignKey("personas.identificador"), primary_key=True)
    numeropais             = Column(Integer)
    verificacionfinanciera = Column(String)
    verificacionjudicial   = Column(String)
    calificacionriesgo     = Column(Integer)
    verificador            = Column(Integer, ForeignKey("empleados.identificador"), nullable=False)
